.github/workflows/workflow.yml

```diff
@@ -447,6 +447,17 @@ jobs:
         run: |
           bash watchdog/e2e-tests/metrics.sh
 
+  reproducibility:
```

Comment:
Should we name this something a bit more descriptive, like `canister-build-reproducibility`?